keyboards/kyria/keymaps/j-inc/keymap.c

```diff
@@ -0,0 +1,334 @@
+/* Copyright 2019 Thomas Baart <thomas@splitkb.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include <stdio.h>
+bool is_alt_tab_active = false;
+uint16_t alt_tab_timer = 0;
+
+char wpm_str[10];
+
+enum layers {
+    _QWERTY = 0,
+    _LOWER = 1,
+    _RAISE = 2,
+    _ADJUST = 3
```

Comment:
You don't actually need the numbers. It starts with 0 and auto-increments. 

```suggestion
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST,
```